model/src/main/java/org/openremote/model/security/KeyStoreService.java

```diff
@@ -14,72 +14,14 @@ public interface KeyStoreService extends ContainerService {
 	String OR_KEYSTORE_PASSWORD = "OR_KEYSTORE_PASSWORD";
 	Logger LOG = Logger.getLogger(KeyStoreService.class.getName());
 
-	KeyStore getKeyStore(KeyStoreType type);
-	void storeKeyStore(KeyStore keyStore, KeyStoreType type);
+	KeyStore getKeyStore();
+	void storeKeyStore(KeyStore keyStore);
+	KeyStore getTrustStore();
+	void storeTrustStore(KeyStore keyStore);
 	char[] getKeyStorePassword();
-
 	KeyManagerFactory getKeyManagerFactory(String preferredAlias) throws Exception;
 	TrustManagerFactory getTrustManagerFactory() throws Exception;
-	/**
-	 * This method is used by all ManagerFactories, that returns True if the provided string s
-	 * @param s The string to check
-	 * @param requestedAlias the alias that the user specified in the client creation. The alias prefixing happens before
-	 *                       this is called.
-	 * @return True, if all criteria are matched.
-	 */
 	static Boolean isRequestedAlias(String s, String requestedAlias){
 		return s.equals(requestedAlias);
 	}
```

Comment:
Don't see why this is needed here

Reply:
I was thinking of using this as a way to allow expandability but at the end of the day I didn't need it. Removed.